tests/load/filesystem/test_sql_client.py

```diff
@@ -297,11 +312,93 @@ def test_delta_tables(destination_config: DestinationTestConfiguration) -> None:
     _run_dataset_checks(
         pipeline,
         destination_config,
+        secret_directory=secret_directory,
         table_format="delta",
         alternate_access_pipeline=access_pipeline,
     )
 
 
+@pytest.mark.essential
+@pytest.mark.parametrize(
+    "destination_config",
+    destinations_configs(all_buckets_filesystem_configs=True, bucket_subset=(AWS_BUCKET,)),
+    ids=lambda x: x.name,
+)
+def test_secrets_management(destination_config: DestinationTestConfiguration) -> None:
```

Comment:
do we need this test in the load? if you move it to `tests/pipelines` it will execute on windows as well. then you'll see if you normalize the paths correctly

Reply:
moved it